shared/chain_support.go

```diff
@@ -29,37 +29,37 @@ type ChainSupport interface {
 }
 
 type CCIPChainSupport struct {
-	lggr            logger.Logger
-	homeChain       reader.HomeChain
-	oracleIDToP2pID map[commontypes.OracleID]libocrtypes.PeerID
-	nodeID          commontypes.OracleID
-	destChain       cciptypes.ChainSelector
+	Lggr            logger.Logger
+	HomeChain       reader.HomeChain
+	OracleIDToP2pID map[commontypes.OracleID]libocrtypes.PeerID
+	NodeID          commontypes.OracleID
+	DestChain       cciptypes.ChainSelector
```

Comment:
Why were these made public?

Reply:
It moved to another package

Reply:
Don't think that explains it. These only need to be exported if they're used by external objects that have access to this object. I don't think thats the case? If it is, something is very wrong.

Reply:
Yes. I didn't change how it was being used in commit. Added `New` function for it now.